packages/bitcoin/src/btc-size-fee-estimator.ts

```diff
@@ -0,0 +1,327 @@
+// https://github.com/argvil19/bitcoin-transaction-size-calculator/blob/master/index.js
```

Comment:
Maybe we can generate some tests for this?

Reply:
Yep, I'll address the lack of code coverage here, thx. 👍